.github/workflows/kubeflow.yml

```diff
@@ -50,21 +62,33 @@ jobs:
           sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
           source $VENV
           zenml integration install kubeflow -y
+          pip install notebook==6.4.11
 
-      - name: Install Linux System Dependencies
-        if: runner.os=='Linux'
-        run: sudo apt install graphviz
+      - name: Perform docker login and get Kubecontext
+        run: bash scripts/setup_kubeflow_aws_connection.sh
 
       - name: Build docker image
         # this step can be much faster if we use an image that copies the
```

Comment:
I'm guessing the current image now uses the lock file, can we remove the comment?